cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -229,6 +229,13 @@ jobs:
 
 {{%- endif %}}
 
+{{%- for job in user_publish_jobs %}}
+  user-publish-jobs:
```

Comment:
This naming is temporary - obviously this won't work in a loop with more than one thing in it. Need to decide how we want these jobs named.

Reply:
This is part of why it seemed reasonable to me for this to be slug-driven (gonna elaborate in a top-level comment).